core/api_backend.go

```diff
@@ -1,30 +1,30 @@
-package rpc
+package core
```

Comment:
I feel `api_backend` is only serving/relevant to rpc (not shared/used by anything else), so putting it into rpc folder makes the code more cohesive.

Reply:
Moving to core package can access more in the package.
It is the functions that core provided to the RPC, so it should be put into the provider (core package).

Reply:
Ah. You're right I confused this with the internal public.go. This one matches better with `eth/api_backend.go`